packages/ethereum/redux/thunks.js

```diff
@@ -52,4 +53,149 @@ export const updateEthereumBalance = ({ tokenSymbol, address, balance, amount })
       return reject(e)
     }
   })
+}
+
+export const selectEthereumWallet = ({ address }) => (dispatch) => {
+  return new Promise((resolve, reject) => {
+    try {
+      dispatch(Actions.selectEthereumWallet({ address }))
+      return resolve()
+    } catch (e) {
+      return reject(e)
+    }
+  })
+}
+
+export const dropEthereumSelectedWallet = () => (dispatch) => {
+  return new Promise((resolve, reject) => {
+    try {
+      dispatch(Actions.dropEthereumSelectedWallet())
+      return resolve()
+    } catch (e) {
+      return reject(e)
+    }
+  })
+}
+
+export const createEthereumTxDraft = ({ address, masterWalletAddress }) => (dispatch) => {
+  return new Promise((resolve, reject) => {
+    try {
+      dispatch(Actions.createEthereumTxDraft({ address, masterWalletAddress }))
+      return resolve()
+    } catch (e) {
+      return reject(e)
+    }
+  })
+}
+
+export const deleteEthereumTxDraft = ({ masterWalletAddress }) => (dispatch) => {
+  return new Promise((resolve, reject) => {
+    try {
+      dispatch(Actions.deleteEthereumTxDraft({ masterWalletAddress }))
+      return resolve()
+    } catch (e) {
+      return reject(e)
+    }
+  })
+}
+
+// export const updateEthereumTxDraftGasPrice = ({ gasPrice, masterWalletAddress }) => (dispatch) => {
+//   return new Promise((resolve, reject) => {
+//     try {
+//       dispatch(Actions.updateEthereumTxDraftGasPrice({ gasPrice, masterWalletAddress }))
+//       return resolve()
+//     } catch (e) {
+//       return reject(e)
+//     }
+//   })
+// }
+
+// export const updateEthereumTxDraftChainId = ({ chainId, masterWalletAddress }) => (dispatch) => {
+//   return new Promise((resolve, reject) => {
+//     try {
+//       dispatch(Actions.updateEthereumTxDraftChainId({ chainId, masterWalletAddress }))
+//       return resolve()
+//     } catch (e) {
+//       return reject(e)
+//     }
+//   })
+// }
```

Comment:
There were my comments. Should we remove them?

Reply:
Yes, the may be removed safely.